pipelines/build-deploy-test-CI-WL.yml

```diff
@@ -2360,13 +2360,14 @@ jobs:
     - get: ops-ui-master
       trigger: true
       passed: ["CI Terraform"]
-    - task: "CI Helm"
-      file: census-rm-deploy/tasks/helm.yml
+    - task: "CI Rabbitmq"
+      file: ((rabbitmq-file))
       params:
         ADMIN_SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
         ENV: ci
         KUBERNETES_CLUSTER: rm-k8s-cluster
         RABBITMQ_CONFIG_VALUES_FILE: rabbitmq/values.yml
```

Comment:
`RABBITMQ_CONFIG_VALUES_FILE` is being passed in but rabbitmq-provision.yml isn't using it.